src/main/java/org/eclipse/golo/compiler/ir/GoloModule.java

```diff
@@ -135,6 +135,20 @@ public void addStruct(Struct struct) {
     struct.setModuleName(getPackageAndClass());
   }
 
+  public GoloElement getSubtypeByName(String name) {
```

Comment:
I'm surprised you did not use the Java stream API :trollface: 

Anyway you are right, this exists early without creating intermediate objects :+1: